gcloud/error_reporting/client.py

```diff
@@ -0,0 +1,243 @@
+        depending on the data that has been provided in the error report.
+    """
+
+    def __init__(self, method=None, url=None,
+                 user_agent=None, referrer=None,
+                 response_status_code=None, remote_ip=None):
+        self.method = method
+        self.url = url
+        # intentionally camel case for mapping to JSON API expects
+        self.userAgent = user_agent
+        self.referrer = referrer
+        self.responseStatusCode = response_status_code
+        self.remoteIp = remote_ip
+
+
+class Client(object):
+    """Error Reporting client. Currently Error Reporting is done by creating
+    a Logging client.
+
+    :type project: string
+    :param project: the project which the client acts on behalf of. If not
+                    passed falls back to the default inferred from the
+                    environment.
+
+    :type credentials: :class:`oauth2client.client.OAuth2Credentials` or
+                       :class:`NoneType`
+    :param credentials: The OAuth2 Credentials to use for the connection
+                        owned by this client. If not passed (and if no ``http``
+                        object is passed), falls back to the default inferred
+                        from the environment.
+
+    :type http: :class:`httplib2.Http` or class that defines ``request()``.
+    :param http: An optional HTTP object to make requests. If not passed, an
+                 ``http`` object is created that is bound to the
+                 ``credentials`` for the current object.
+
+    :type service: str
+    :param service: An identifier of the service, such as the name of the
+                    executable, job, or Google App Engine service name. This
+                    field is expected to have a low number of values that are
+                    relatively stable over time, as opposed to version,
+                    which can be changed whenever new code is deployed.
+
+
+    :type version: str
+    :param version: Represents the source code version that the developer
+                    provided, which could represent a version label or a Git
+                    SHA-1 hash, for example. If the developer did not provide
+                    a version, the value is set to default.
+
+    :raises: :class:`ValueError` if the project is neither passed in nor
+             set in the environment.
+    """
+
+    def __init__(self, project=None,
+                 credentials=None,
+                 http=None,
+                 service=None,
+                 version=None):
+        self.logging_client = gcloud.logging.client.Client(
+            project, credentials, http)
+        self.service = service if service else self.DEFAULT_SERVICE
+        self.version = version
+
+    DEFAULT_SERVICE = 'python'
+
+    def _send_error_report(self, message,
+                           report_location=None, http_context=None, user=None):
+        """Makes the call to the Error Reporting API.
```

Comment:
Ok I added some more text about that to the docstring.

The idea is this method will be entirely replaced by a call to the API but I agree we should make it clear what's going on in the meantime.